### DISCUSSION TITLE
Could not register to oast.me

### DISCUSSION THREAD

**Discussion by www1759896801** [projectdiscovery/nuclei#2211] [Q&A] [answered]
Could not register to oast.me: could not make register request: POST https://oast.site/register giving up after 1 attempts: Post "https://oast.site/register": dial tcp: lookup oast.site on 192.168.78.2:53: read udp 192.168.78.131:35957->192.168.78.2:53: i/o timeout, retrying with remaining

**Comment by www1759896801** [comment]
why

**Comment by ehsandeep** [comment, marked as answer]
@www1759896801 as the error indicates `dial tcp: lookup oast.site on 192.168.78.2:5`, there is some network issue on your local system and you can confirm interactsh servers are accessible and working fine here - https://status.projectdiscovery.io/

**Reply by www1759896801** [reply]
thank you，太感谢您了
